test/streaming_engine_integration.js

```diff
@@ -522,6 +522,7 @@ describe('StreamingEngine', function() {
    * Initializes or switches to the given period.
    *
    * @param {shakaExtern.Period} period
+   * @return {{audio: string, video: string}}
```

Comment:
Not sure this is right.  The values in this dictionary are videoStream and audioStream vars.  So those should be Streams, I think.